internal/util/projutil/exec.go

```diff
@@ -49,6 +49,8 @@ type GoCmdOptions struct {
 	Env []string
 	// Dir is the dir to run "go {cmd}" in; exec.Command.Dir is set to this value.
 	Dir string
+	// Race is to add the -race flag when compiling the operator
```

Comment:
An extra flag is not needed. `-race` can be appended directly to `GoCmdOptions.Args` in the caller.